Documentation/Commands/ib.md

```diff
@@ -0,0 +1,19 @@
+# Interface Builder parser
+
+## Input
+
+The interface builder parser accepts either a file or a directory, which it'll search for `storyboard` files. The parser will load all the scenes and all the segues for each storyboard. It supports both AppKit (macOS) and UIKit (iOS, watchOS, tvOS) storyboards.
+
+### Filter
+
+The default filter for this command is: `[^/]\.storyboard$`. That means it'll accept any file with the extension `storyboard`.
```

Comment:
At some point we discussed about keeping the (now deprecated) `storyboards` subcommand and give it a default filter of `.storyboard$` while the `ib` subcommand was gonna default to `.(xib|storyboard)$`… do we still want to do that? Or will we just switch the default filter for `ib` when we start spring XIBs (for reuseIdentifiers and stuff) and continue deprecating the storyboard command?

Reply:
Depends on when we get `xib` support in. If it's not ready by 7.0, we may as well drop the alias 🤷‍♂️